controller/rest/auth.go

```diff
@@ -662,7 +666,7 @@ func restReq2User(r *http.Request) (*loginSession, int, string) {
 
 	// Check token end-of-life
 	now := time.Now()
-	if now.Unix() >= claims.ExpiresAt {
+	if claims.ExpiresAt != nil && now.After(claims.ExpiresAt.Time) {
```

Comment:
We should reject the claims without `ExpiresAt`, otherwise this JWT token will never expire.